### DISCUSSION TITLE
Saved filters in "console.yaml" file

### DISCUSSION THREAD

**Discussion by Tito-aix** [akvorado/akvorado#1571] [Q&A] [answered]
Hi Vincent, first of all thanks for your amazing work on this project !
I have a question regarding "console.yaml" file and "saved-filters" category :
Even if I delete all the descriptions & contents inside the file, and even after restarting the containers, I still have my historical filter in the "Visualize" tab.
Could you please help me to purge and start with new ones ?
Also, could you please give me the correct syntax regarding "homepage-graph-timerange", I've tried "1", "1 hour", "3600" but without success.
I'm currently working on akvorado version 1.11.1.

**Comment by vincentbernat** [comment, marked as answer]
You need to restart the orchestrator container as well (this is the one distribution the configuration)

> Durations must be written using strings like `10h20m` or `5s`. Valid time units are `ms`, `s`, `m`, and `h`.

**Reply by Tito-aix** [reply]
Vincent,
Many thanks for your quick response.
I'll try durations regarding homepage-graph-timerange.
Regarding the saved filters, I've already restarted the orchestrator container several times with docker restart ID, also docker compose stop and then up, and even rebooted my VM.
All the filters are still present even if console.yaml is empty.

**Reply by Tito-aix** [reply]
Hi Vincent,
The "1h" setting for homepage-graph-timerange is working perfectly after VM reboot.
I've finally handled the "saved-filter" issue by removing all the native filters (Netflix + GAFAM) and reboot the VM.
We can close this thread !
I'm only doing research & development regarding open-source network monitoring tools and going to push my work by the end of this week, but I'm gonna follow your changelogs and update for sure !
Thanks again for your help.
